src/cli.ts

```diff
@@ -1,9 +1,30 @@
 #!/usr/bin/env node
 
-import { checkAndPreparePackage } from './index.js'
+import * as fs from 'node:fs'
 
-void checkAndPreparePackage(
-  process.argv[2],
-  process.argv[3],
-  ['1', 'check', 'true', 'yes'].includes(process.argv[4]),
-)
+import { PACKAGE_JSON } from './constants'
+
+import { checkAndPreparePackage, PackageJson } from './index.js'
+
+function readCwdPackage(packageName: string): PackageJson | undefined {
+  try {
+    const packageJson = JSON.parse(
+      fs.readFileSync(PACKAGE_JSON, 'utf8'),
+    ) as PackageJson
+    if (packageJson.name === packageName) {
```

Comment:
I think `packageJson.version` should also be checked? `checkVersion` is for sub packages.

Reply:
It feels an unlikely scenario for them not to match, but doesn't hurt to be thorough :) Fixed